docs/references/api.rst

```diff
@@ -34,7 +35,6 @@ PostgREST exposes three database objects of a schema as resources: tables, views
       '#casting-columns': 'api/tables_views.html#casting-columns',
       '#json-columns': 'api/tables_views.html#json-columns',
       '#composite-array-columns': 'api/tables_views.html#composite-array-columns',
-      '#computed-virtual-columns': 'api/tables_views.html#computed-virtual-columns',
```

Comment:
Hmm... maybe this should be changed instead of deleted. It avoids the double redirect and goes directly to `computed_fields.html`.

```javascript
      '#computed-virtual-columns': 'api/computed_fields.html#computed-fields',
```